infrastructure/terraform/terraform-sample.tfvars

```diff
@@ -27,25 +25,25 @@ deploy_pipelines     = true
 ####################  DATA VARIABLES  #################################
 
 data_project_id              = "Project id where the MDS datasets will be created"
+destination_data_location    = "BigQuery location (either regional or multi-regional) for the MDS BigQuery datasets."
 data_processing_project_id   = "Project id where the Dataform will be installed and run"
 source_ga4_export_project_id = "Project id which contains the GA4 export dataset"
-source_ga4_export_dataset    = "GA4 export dataset name"
-source_ads_export_data = [{ project = "abc", dataset = "dataset1", table_suffix = "_123456" },
-{ project = "xyz", dataset = "dataset2", table_suffix = "_567890" }]
+source_ga4_export_dataset    = "GA4 export dataset name. Do not include the project id, just the name."
+source_ads_export_data       = [
+  { project = "abc", dataset = "dataset1", table_suffix = "_123456" },
+  { project = "xyz", dataset = "dataset2", table_suffix = "_567890" }
+]
 
 ####################  ACTIVATION VARIABLES  #################################
 
-activation_project_id = "Project ID where activation resources are created"
+activation_project_id  = "Project ID where activation resources will be created"
 # Required. A MEASUREMENT ID and API SECRET generated in the Google Analytics UI. To create a new secret, navigate to:
 #   Admin > Data Streams > choose your stream > Measurement Protocol > Create
 ga4_measurement_id     = "Measurement ID in GA4"
-ga4_measurement_secret = "Client secret for authenticatin to GA4 API"
+ga4_measurement_secret = "Client secret for authentication to GA4 API"
 
 ####################  GITHUB VARIABLES  #################################
 
 project_owner_email   = "Project owner email"
-dataform_github_repo  = "URL of the GitHub or GitLab repo which contains the Dataform scripts"
+dataform_github_repo  = "URL of the GitHub or GitLab repo which contains the Dataform scripts. Should start with https://"
 dataform_github_token = "GitHub token generated for that repo"
-
-pipelines_github_owner = "Cloud Build github owner account for pipelines"
-pipelines_github_repo  = "Cloud Build github repository for pipelines"
```

Comment:
pipelines_github_owner and pipelines_github_repo are use, can not be deleted

Reply:
I provided fake default values in variables.tf